src/components/Counter/Counter.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import "./styles.css";
+
+class Counter extends Component {
+  state = {
+    value: 0,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state.value = props.value;
```

Comment:
Let's rename props.value to props.**initialValue**. 

https://ru.reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#recommendation-fully-uncontrolled-component-with-a-key

Reply:
Done (46f3b898fc701ef3a4d0fd3315d20036399fe336)